Tutorial.md

```diff
@@ -11,477 +10,247 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
 > + [API Blueprint Map][map]
 > + [Tools for API Blueprint][Tooling Section]
 
-## Gist Fox API
-Without any further ado our *Gist Fox API* blueprint starts like this:
+## API Blueprint
 
-    FORMAT: 1A
+The start of the blueprint looks as follows:
 
-    # Gist Fox API
-    Gist Fox API is a **pastes service** similar to [GitHub's Gist](http://gist.github.com).
+```markdown
+FORMAT: 1A
 
-What have we just done? Let's break it line by line:
+# Polls
 
-#### Metadata
+Polls is a simple API allowing consumers to view polls, and vote in them.
+```
 
-    FORMAT: 1A
+## Metadata
 
-Depending on the tools you will be using with your blueprint you might want to start your blueprint file with a metadata explicitly stating the API Blueprint version you want to use like in this case the version 1A.
+The blueprint starts wih a metadata section. In this case we have specified that `FORMAT` has the value of `1A`. The format keyword specifies the version of the API Blueprint. In this case 1A.
 
-> **Note:** Should you need a clarification of some terms as used through this document refer to [API Blueprint Glossary of Terms][].
+## API Name & Description
 
-#### API Name
+The first heading in the blueprint serves as the name of your API, which in this case is "Polls". Headings start with one or more `#` symbols followed by a title. The API Name here uses one hash to distinguish it as the first level. The number of `#` you use will determine the size of the heading.
```

Comment:
Move period within quotation marks.

> ...which in this case is "Polls."

Also, is it the "size of the heading" or is it the level of the heading (or something similar)?